GOVERNANCE.md

```diff
@@ -128,10 +135,71 @@ the issue tracker is:
   either the proposal is dropped or the objecting members are persuaded. If
   there is an extended impasse, a motion for a vote may be made.
 
+## Collaborator Nominations
+
+Any existing Collaborator can nominate an individual making significant
+and valuable contributions across the Node.js organization to become a new
+Collaborator.
+
+To nominate a new collaborator, open an issue in the [nodejs/node][]
+repository, with a summary of the nominee's contributions, including but
+not limited to:
+
+* Commits in the [nodejs/node][] repository.
+  * Can be shown using the link
+    `https://github.com/nodejs/node/commits?author=${GITHUB_ID}`
+    (replace `${GITHUB_ID}` with the nominee's GitHub ID).
+* Pull requests and issues opened in the [nodejs/node][] repository.
+  * Can be shown using the link
+    `https://github.com/nodejs/node/pulls?q=author%3A${GITHUB_ID}+`
+* Comments and reviews on issues and pull requests in the
+  [nodejs/node][] repository
+  * Can be shown using the links
+    `https://github.com/nodejs/node/pulls?q=reviewed-by%3A${GITHUB_ID}+`
+    and `https://github.com/nodejs/node/pulls?q=commenter%3A${GITHUB_ID}+`
+* Assistance provided to end users and novice contributors
+* Participation in other projects, teams, and working groups of the
+  Node.js organization
+  * Can be shown using the links
+    `https://github.com/search?q=author%3A${GITHUB_ID}++org%3Anodejs&type=Issues`
+    and
+    `https://github.com/search?q=commenter%3A${GITHUB_ID}++org%3Anodejs&type=Issues`
+* Other participation in the wider Node.js community
+
+Mention @nodejs/tsc and @nodejs/collaborators in the issue to notify the TSC
+and other collaborators about the nomination.
+
+If there are no objections raised by any collaborators one week after
+the issue is opened, the nomination will be considered as accepted.
+Should there be any objections against the nomination, the TSC is responsible
+for working with the individuals involved and finding a resolution.
+The nomination must be approved by the TSC, which is assumed when there are no
+objections from any TSC members.
+
+Prior to the public nomination, the collaborator initiating it can seek
+feedback from other collaborators in private using
+[the GitHub discussion page][collaborators-discussions] of the
+collaborators team, and work with the nominee to improve the nominee's
+contribution profile, in order to make the nomination as frictionless
+as possible.
+
+If individuals making valuable contributions do not believe they have been
+considered for a nomination, they may log an issue or contact a collaborator
+directly.
+
+### Onboarding
+
+When the nomination is accepted, the new Collaborator will be onboarded
```

Comment:
"Collaborator" is capitalized here but not elsewhere in the new text (such as the previous paragraph). We seem to have standardized on capitalizing it, so I'd recommend doing that throughout the doc, or at least in the new text.